resource_booking/tests/test_backend.py

```diff
@@ -268,9 +268,9 @@ def test_booking_from_calendar_view(self):
         self.assertEqual(self.rbt.duration, 0.5)
         # Change it to 45 minutes
         self.rbt.duration = 0.75
-        # Bookings smart button configures calendar with slots of 45 minutes
+        # Bookings smart button configures calendar with slots from slot duration field
         button_context = self.rbt.action_open_bookings()["context"]
-        self.assertEqual(button_context["calendar_slot_duration"], "00:45")
+        self.assertEqual(button_context["calendar_slot_duration"], "00:30:00")
```

Comment:
Why this change in the test?

Reply:
All this change is related to the `slot_duration` field that is added in `resource.booking.type`, it changes with respect to how it is set in v15: https://github.com/OCA/calendar/blob/15.0/resource_booking/models/resource_booking_type.py#L179 Is it necessary? I think not because there are now 2 fields to reflect the same thing.

Reply:
Well, that slot is though for presenting the calendar in intervals of such duration, so it has a goal, but not sure how it was handled before and now.

Reply:
In v15 this already exists (with the `duration` field), so I comment that I don't understand the need for this extra field for "the same thing".

Reply:
Ok, I see that it is related to https://github.com/OCA/calendar/pull/101#discussion_r1374273498 and is intentional.